server/controllers/middlewares/auth.js

```diff
@@ -0,0 +1,12 @@
+const jwt = require('jsonwebtoken');
+
+module.exports = (req, res, next) => {
+  jwt.verify(req.cookies.hotmeal_token, process.env.PRIVATEKEY, (err, token) => {
```

Comment:
What if there are no cookies at all? 
It does not make sense to verify undefined, right? So check if there is a cookie in the request header and if yes, verify it. If no, throw 401 error

Good job, guys :)